CIAC-12379 - Align Zoom Pack docs (#38044)
demisto/content: Packs/Zoom/Scripts/ZoomAsk/README.md

![SlackAsk](../../doc_files/66044107-7de39f00-e529-11e9-8099-049502b4d62f.png)

The automation can utilize the interactive capabilities of Zoom to send a form with buttons. 
This requires the external endpoint for interactive responses to be available for connection. (See the [Zoom integration documentation](https://xsoar.pan.dev/docs/reference/integrations/zoom)).
You can also utilize a dropdown list instead, by specifying the `responseType` argument.

> **In order to use `ZoomAsk` via playbook**:
> - Add the `ZoomAsk` script to a playbook as a task.
> - In the `message` argument, specify the message to be sent.
> - Configure the response options by filling out the `option1` and `option2` arguments (default values are 'Yes' and 'No').
> - Either a user or a channel_id or channel_name must be specified.
> - In the `ZoomAsk` task, pass a tag value to the `task` argument.
> - All other inputs are optional.
> - At some point at the playbook, after running `ZoomAsk`, add a manual conditional task, which holds up the playbook execution until the response is received from Zoom.
> - The condition names must match the response options you passed in to `ZoomAsk`.
> - In order to tie the conditional task back to `ZoomAsk`:
>     - Add the same tag from the fourth step to the conditional task (under the "Details" tab of the task).
> - The conditional task will be marked as completed when a user responds to the `ZoomAsk` form.

Reviewer: ```suggestion
To use `ZoomAsk` via playbook:
1. Add the `ZoomAsk` script to a playbook as a task.
2. In the `message` argument, specify the message to be sent.
3. Configure the response options by filling out the `option1` and `option2` arguments (default values are 'Yes' and 'No').
4. Either a user or a channel_id or channel_name must be specified.
5. In the `ZoomAsk` task, pass a tag value to the `task` argument.

All other inputs are optional.

At some point at the playbook, after running `ZoomAsk`, add a manual conditional task, which holds up the playbook execution until the response is received from Zoom.

The condition names must match the response options you passed in to `ZoomAsk`.

In order to tie the conditional task back to `ZoomAsk`, add the same tag from the fourth step to the conditional task (under the "Details" tab of the task). The conditional task will be marked as completed when a user responds to the `ZoomAsk` form.
```

> This requires the external endpoint for interactive responses to be available for connection. (See the Zoom integration documentation for more information).

Reviewer 2: ```suggestion
This requires the external endpoint for interactive responses to be available for connection. (See the [Zoom integration documentation](https://xsoar.pan.dev/docs/reference/integrations/zoom) for more information).
```

You can also utilize a dropdown list instead, by specifying the `responseType` argument.

> In order to use ZoomAsk via playbooks:

Reviewer 2: ```suggestion
In order to use `ZoomAsk` via playbook:
```

>     1. Add the ZoomAsk script to a playbook as a task.

Reviewer 2: ```suggestion
    1. Add the `ZoomAsk` script to a playbook as a task.
```

>     2. In the 'Message' argument, specify the message to be sent.

Reviewer 2: ```suggestion
    2. In the `message` argument, specify the message to be sent.
```

>     3. Configure the response options by filling out the 'Option1' and 'Option2' arguments (default values are 'Yes' and 'No').

Reviewer 2: ```suggestion
    3. Configure the response options by filling out the `option1` and `option2` arguments (default values are 'Yes' and 'No').
```

    4. Either a user or a channel_id or channel_name must be specified.
    5. All other inputs are optional.

>     6. At some point at the playbook, after running ZoomAsk, add a mutual conditional task which holds up the playbook execution until the response is received form zoom.

Reviewer 2: ```suggestion
    6. At some point at the playbook, after running `ZoomAsk`, add a manual conditional task, which holds up the playbook execution until the response is received from Zoom.
```

>        1. The condition names should much the response options you passed in to the ZoomAsk.

Reviewer 2: ```suggestion
       1. The condition names must match the response options you passed in to `ZoomAsk`.
```

>        2. In order to tie the conditional task back to ZoomAsk:

Reviewer 2: ```suggestion
       2. In order to tie the conditional task back to `ZoomAsk`:
```

>           - Add a tag to the conditional task.

Reviewer 2: ```suggestion
          - Add a tag to the conditional task (under the "Details" tab of the task).
```

>           - In the ZoomAsk task, include the same tag value under the "task" argument.

Reviewer 2: ```suggestion
          - In the `ZoomAsk` task, pass in the same tag value to the `task` argument.
```

Reviewer 2: I would change the order here: I'd move this step up to right after step 4 when you are listing all the arguments that must be passed in to `ZoomAsk`.

>     7. The conditional task remark as completed when executing the playbook.

Reviewer 2: ```suggestion
    7. The conditional task will be marked as completed when a user responds to the `ZoomAsk` form.
```

> 

Reviewer 2: For some reason when you preview this file, this section all shows up on one line (maybe because of the indent?) Suggest changing to one list item per line instead.
